inc/update.class.php

```diff
@@ -439,6 +439,11 @@ public function doUpdates($current_version = null) {
             update93to94();
             break;
 
+         case "9.4.0":
```

Comment:
Should we add this ?
```suggestion
         case "9.4.0":
         case "9.4.1-dev":
```

Reply:
Nope, this would be currently terribly annonying ;)